demos/pedestrian_tracker_demo/cpp/include/pedestrian_tracker_demo.hpp

```diff
@@ -40,6 +40,18 @@ static const char delay_message[] = "Optional. Delay between frames used for vis
 static const char output_log_message[] = "Optional. The file name to write output log file with results of pedestrian tracking. "
                                           "The format of the log file is compatible with MOTChallenge format.";
 static const char utilization_monitors_message[] = "Optional. List of monitors to show initially.";
+static const char at_message[] = "Required. Architecture type: centernet, ssd or yolo";
```

Comment:
```suggestion
static const char at_message[] = "Required. Architecture type for detector model: centernet, ssd or yolo";
```
The same for nireq, nstreams, nthreads parameters. In demos with two models it is not obvious, for which model these options can be set.

Reply:
done